docs/NETWORKING.md

```diff
@@ -0,0 +1,156 @@
+# EVE Networking (Overview)
```

Comment:
Thank you for this overview document.

With it plus APP-CONNECTIVITY.md it seems like we've subsumed (with more details) what's in NETWORK-MODELS.md. Can you check whether that is the case and if so remove it?

Reply:
True, I checked and there is now no added value in NETWORK-MODELS.md
I have removed the file and links to it.